tile-generation/src/main/scala/com/oculusinfo/tilegen/tiling/TileOperations.scala

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright (c) 2014 Oculus Info Inc.
+ * http://www.oculusinfo.com/
+ *
+ * Released under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+ * of the Software, and to permit persons to whom the Software is furnished to do
+ * so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.oculusinfo.tilegen.tiling
+
+import java.lang.{Double => JavaDouble}
+import java.text.SimpleDateFormat
+
+import com.oculusinfo.binning.impl.AOITilePyramid
+import com.oculusinfo.binning.util.JsonUtilities
+import com.oculusinfo.tilegen.datasets.SchemaTypeUtilities._
```

Comment:
It's a very minor thing, but...

I have a gut feeling that it's a bad idea to import objects (like you're doing here for SchemaTypeUtilities and SparkSqlUtils) at such a high level - i.e., in unbounded scope.

I would prefer them imported, say, for a given class, as the first line in that class.

I'm happy to be overruled - if, say, you don't like imports inside classes for some reason, but would I think a small discussion on it might be warranted.